### PR TITLE
Update Set-CsOnlineLisPort.md

### PR DESCRIPTION
**Pull request by danny-levin**
Added 2 examples

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsOnlineLisPort.md [hunk 1]
```diff
@@ -36,6 +36,18 @@ Set-CsOnlineLisPort -PortID 12174 -ChassisID 0B-23-CD-16-AA-CC -Description "Lis
 ```
 
 Example 1 creayes the association between port "12174" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".
+### -------------------------- Example 2 --------------------------
+```
+Set-CsOnlineLisPort -PortID 0A-25-55-AB-CD-FF -ChassisID 0B-23-CD-16-AA-CC -Description "LisPort 0A-25-55-AB-CD-FF" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
+```
+
+Example 2 creayes the association between port "0A-25-55-AB-CD-FF" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsOnlineLisPort.md:44]:
```suggestion
Example 2 creates the association between port "0A-25-55-AB-CD-FF" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".
```

File: skype/skype-ps/skype/Set-CsOnlineLisPort.md [hunk 2]
```diff
@@ -36,6 +36,18 @@ Set-CsOnlineLisPort -PortID 12174 -ChassisID 0B-23-CD-16-AA-CC -Description "Lis
...
+### -------------------------- Example 3 --------------------------
+```
+Set-CsOnlineLisPort -PortID 12174 -ChassisID 55123 -Description "LisPort 12174" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
+```
+
+Example 3 creayes the association between port "12174" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee". (Note: in this example, ChassisID sub-type  is InterfaceName)
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsOnlineLisPort.md:50]:
```suggestion
Example 3 creates the association between port "12174" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee". (Note: in this example, ChassisID sub-type  is InterfaceName) 
```

File: skype/skype-ps/skype/Set-CsOnlineLisPort.md [hunk 3]
```diff
@@ -36,6 +36,18 @@ Set-CsOnlineLisPort -PortID 12174 -ChassisID 0B-23-CD-16-AA-CC -Description "Lis
 ```
 
 Example 1 creayes the association between port "12174" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".
+### -------------------------- Example 2 --------------------------
+```
+Set-CsOnlineLisPort -PortID 0A-25-55-AB-CD-FF -ChassisID 0B-23-CD-16-AA-CC -Description "LisPort 0A-25-55-AB-CD-FF" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsOnlineLisPort.md:41]:
```suggestion
PS C:\> Set-CsOnlineLisPort -PortID 0A-25-55-AB-CD-FF -ChassisID 0B-23-CD-16-AA-CC -Description "LisPort 0A-25-55-AB-CD-FF" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
```

File: skype/skype-ps/skype/Set-CsOnlineLisPort.md [hunk 4]
```diff
@@ -36,6 +36,18 @@ Set-CsOnlineLisPort -PortID 12174 -ChassisID 0B-23-CD-16-AA-CC -Description "Lis
...
+```
+
+Example 2 creayes the association between port "0A-25-55-AB-CD-FF" and LocationId "efd7273e-3092-4a56-8541-f5c896bb6fee".
+### -------------------------- Example 3 --------------------------
+```
+Set-CsOnlineLisPort -PortID 12174 -ChassisID 55123 -Description "LisPort 12174" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsOnlineLisPort.md:47]:
```suggestion
PS C:\> Set-CsOnlineLisPort -PortID 12174 -ChassisID 55123 -Description "LisPort 12174" -LocationId efd7273e-3092-4a56-8541-f5c896bb6fee
```

**Comment by get-itips** [comment]
agree with @dariomws

**Comment by danny-levin** [comment]
Suggested change is the same as current text, hence I am not clear what change is requested

**Comment by dariomws** [comment]
Hi @danny-levin, thank you for your help to improve docs.microsoft.com.

The most important change I'm suggesting is to replace "creayes" with "creates". I think it's a typo in the first example.

Also, I'm suggesting to add "PS C:\> " to each command. You should have the option to commit these updates.

**Comment by danny-levin** [comment]
Thanks for catching it.  committed requested changes

**Comment by dariomws** [comment]
@danny-levin Thanks again, for taking out some time to create this pull request. I appreciate and encourage you to do the same in the future also.

Please, don´t close it as it needs to be merged to appear in docs.microsoft.com. Thanks!

**Comment by danny-levin** [comment]
Thanks a lot for catching it.
I added 2 examples for using various sub types.
I cut & paste the first example and changed the types.
I assume that the typo was before but I didn’t notice…

BTW the 2 examples that I added are not shown here  https://docs.microsoft.com/en-us/powershell/module/skype/set-csonlinelisport?view=skype-ps
When will they be published?

**Comment by danny-levin** [comment]
Thanks for your help😊 – I am new to editing exiting docs.
What do I need to do to complete this process?

**Comment by dariomws** [comment]
@danny-levin you're welcome. When the PR is merged, you'll see the updates in docs.microsoft.com after a few days.

**Comment by yogkumgit** [comment]
@chrisda Please Merge this PR. Thanks

**Pull request merged**